edu.berkeley.BOINC.yml

```
@@ -0,0 +1,81 @@
+app-id: edu.berkeley.BOINC
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: boincmgr
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```
Unless the app doesn't support wayland, in which case the wayland socket shall be removed

---

Right, the simulation graphics don't work on wayland so I guess I'll remove that.